apps/e2e/split-view-storybook/.storybook/preview.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  previewWrapperDecorators,
+  previewWrapperGlobalTypes,
+  previewWrapperParameters,
+} from '@skyux/storybook';
+import { moduleMetadata } from '@storybook/angular';
```

Comment:
Every other storybook project had this content in the preview file - this did not for some reason. I added it so that we would get the providers that the preview pulls in for us.